graph/step.go

```diff
@@ -141,3 +143,9 @@ func (s *Step) IsBuildStep() bool {
 func (s *Step) IsPushStep() bool {
 	return len(s.Push) > 0
 }
+
+func (s *Step) UpdateBuildStepWithDefaults() {
+	if s.IsBuildStep() && runtime.GOOS == "windows" && !strings.Contains(s.Build, "--isolation") {
+		s.Build = fmt.Sprintf("--isolation hyperv %s", s.Build)
```

Comment:
shouldn't this be `%s --isolation hyperv`?

Reply:
Doesn't the docker build command end with context? s.Build doesn't include "docker build". 

docker build [OPTIONS] PATH | URL | -

Reply:
Oh right, so it doesn't matter here actually.